azure-cli/functionapp/deploy.ps1

```diff
@@ -69,7 +67,7 @@ $output = az functionapp create `
   --name $functionName `
   --resource-group $resourceGroupName `
   --storage-account $storageAccountId `
-  --app-insights-key $instrumentationKey `
+  --app-insights $insightsName `
```

Comment:
From docs: 
Name of the existing App Insights project to be added to the function app. Must be in the same resource group.

Reply:
@lupusbytes @perkops FYI